Documentation/install/upgrade.rst

```diff
@@ -243,6 +243,18 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 
 * Cilium has bumped the minimal kubernetes version supported to v1.11.0.
 
+* The ``kubernetes.io/cluster-service`` label has been removed from the Cilium
+  `DaemonSet`. Existing users must remove this label from their `DaemonSet`
+  specification to safely upgrade.
+
+  *Required action:*
```

Comment:
You can write that this is not required because users can upgrade as long they manually keep adding the label during the upgrades. But having the label makes cilium being restarting in self-managed k8s (point k8s upstream issue)

Reply:
"Highly recommended"?

Reply:
What concerns me is to try to ensure that unless there's a really experienced user or a really good reason, everyone should run the same YAMLs. Just simplifies supporting users.